lib/team.rb

```diff
@@ -1,8 +1,8 @@
 class Team
   attr_reader :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :link
   def initialize(row)
-    @team_id = row["team_id"].to_i
-    @franchise_id = row["franchiseId"].to_i
+    @team_id = row["team_id"]
+    @franchise_id = row["franchiseId"]
```

Comment:
This data type will be different for one of the branches.  We can change that later.